### PR TITLE
Add command for polling

### PR DESCRIPTION
**Pull request by tianhongtan**
polling is fun my dudes

tests:
https://imgur.com/a/yWEK4L2

### CODE REVIEW

**Comment by tianhongtan** [comment]
Will change the emojis from 1-10 to just be random channel emotes

File: commands/poll.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const { prototype } = require("../modules/DiscordClientAwareScheduler");
```
Comment (mynameisivan) [inline commands/poll.js:1]: minor: why do u need this scheduler for poll

File: modules/ArgParser.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const magic = /[^\s"]+|"([^"]*)"/gi;
```
Comment (mynameisivan) [inline modules/ArgParser.js:1]: nice! i should incorp this to roles

**Review by mynameisivan — APPROVED** [review]

File: commands/poll.js (hunk 1 above)
Comment (I-AM-STEVEe) [reply commands/poll.js:1]: cuz copy paste lolol

File: modules/ArgParser.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+/**
+ * discord.js args are strictly split by space. This handles the scenario where you want to have 
+ * multi word arguments wrapped by quotes. e.g.
+ *    "some long first input" second "third input" 
+ *    becomes: 
+ *      ["some long first input", "second", "third"]
```
Comment (I-AM-STEVEe) [inline modules/ArgParser.js:8]: should be "third input" right?

**Review by I-AM-STEVEe — APPROVED** [review]
good stuffs

**Pull request merged**